app/src/main/java/org/mozilla/fenix/settings/SearchEngineListPreference.kt

```diff
@@ -72,7 +76,14 @@ abstract class SearchEngineListPreference : Preference, CoroutineScope {
             return
         }
 
-        val defaultSearchEngine =
+        // To get the default search engine we have to pass in a name that doesn't exist
+        // https://github.com/mozilla-mobile/android-components/issues/3344
+        val defaultSearchEngine = context.components.search.searchEngineManager.getDefaultSearchEngine(
+            context,
+            "."
```

Comment:
This should be a named constant value. Is a period something a user could enter as valid input?

Reply:
@colintheshots This is a hack (detailed in the comment). Wouldn't a constant hide it even more?